pkg/storaged/stratis/pool.jsx

```diff
@@ -120,29 +121,7 @@ function create_fs(pool) {
                                { title: _("Custom mount options"), tag: "extra", type: "checkboxWithInput" },
                            ]
                        }),
-            SelectOne("at_boot", _("At boot"),
-                      {
-                          value: client.in_anaconda_mode() ? "local" : "nofail",
-                          explanation: mount_explanation.nofail,
-                          choices: [
-                              {
-                                  value: "local",
-                                  title: _("Mount before services start"),
-                              },
-                              {
-                                  value: "nofail",
-                                  title: _("Mount without waiting, ignore failure"),
-                              },
-                              {
-                                  value: "netdev",
-                                  title: _("Mount after network becomes available, ignore failure"),
-                              },
-                              {
-                                  value: "never",
-                                  title: _("Do not mount"),
-                              },
-                          ]
-                      }),
+            at_boot_input(client.in_anaconda_mode() ? "local" : "nofail"),
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19849-20240117-090134-2dfecb5e-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)